Documentation/teaching/lectures/interrupts.rst

```diff
@@ -176,26 +177,46 @@ I/O APIC is used to distribute IRQ from external devices to CPU cores.
 
 After discussing the hardware, now let's see how the processor handles an interrupt.
 
+Interrupt control
+-----------------
+
+In order to synchronize the access to shared data between the interrupt handler
+and other potential concurrent activities such as driver initialization or
+driver data processing, it is often required to enable and disable interrupts in
+a controlled fashion.
+
+This can be accomplished at several levels:
+
 .. slide:: Enabling/disabling the interrupts
    :inline-contents: True
    :level: 2
 
+   * at the device level
```

Comment:
at device level

Reply:
hmm I've asked a language person and she said at the device level; also what google says ("at device level" -> 67K results, "at the device level" -> 330K results)